editor/editor_themes.cpp

```diff
@@ -332,11 +332,14 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	style_default->set_draw_center(true);
 
 	// Button and widgets
-	Ref<StyleBoxFlat> style_widget = style_default->duplicate();
+	const float extra_spacing = EDITOR_DEF("interface/theme/additional_spacing", 0.0);
 
+	Ref<StyleBoxFlat> style_widget = style_default->duplicate();
+	style_widget->set_default_margin(MARGIN_LEFT, (extra_spacing + default_margin_size + 6) * EDSCALE);
```

Comment:
+default_margin_size? why, also these kind of changes make the editor_themes more error prone, it is easy to forget to add extra spacing when doing changes. in any case add extra_spacing to default_margin_size, but again so it scales everything consistently

Reply:
yea this was a mistake. as you see just fixed it ;)

Reply:
don't know how that got there. I did some weird git stuff for rebasing on master which. wanted to grab the changes above another branch... probably I broke it there...